site/content/en/latest/contributions/design/backend.md

```diff
@@ -0,0 +1,151 @@
+
+Common use cases for non-Service backends in the K8s and Envoy ecosystem include:
+- Cluster-external endpoints, which are currently second-class citizens in Gateway-API 
+  (supported using [Services and FQDN endpoints][]).
+- Host-local endpoints, such as sidecars or daemons that listen on [unix domain sockets][] or envoy [internal listeners][], 
+  that cannot be represented by a K8s service at all.
+
+Several projects currently support backends that are not registered in the infrastructure-specific service registry. 
+- K8s Ingress: [Resource Backends][]
+- Istio: [Service Entry][]
+- Gloo Edge: [Upstream][]
+- Consul: [External Services][]
+
+## Goals
+* Add an API definition to hold settings for configuring Unix Domain Socket, FQDN and IP.
+* Determine which resources may reference the new backend resource.
+* Determine which existing Gateway-API and Envoy Gateway policies may attach to the new backend resource. 
+
+## Non Goals
+* Support specific backend types, such as S3 Bucket, Redis, AMQP, InfluxDB, etc.  
+
+## Implementation
+
+The `Backend` resource is an implementation-specific Gateway-API [BackendObjectReference Extension][]. 
+
+### Example
+Here is an example highlighting how a user can configure a route that forwards traffic to both a Service and a local
+unix domain socket.
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: backend-svc
+spec:
+  ports:
+    - name: http
+      port: 3000
+      targetPort: 3000
+  selector:
+    app: backend
+---
+apiVersion: gateway.envoyproxy.io/v1alpha1
+kind: Backend
+metadata:
+  name: backend-mixed-ip-uds
+spec:
+  addresses:
+    - unixDomainSocketAddress:
+        path: /var/run/backend.sock
+      applicationProtocol: HTTP2
+      name: uds-be
+    - socketAddress:
+        address: 10.244.0.28
+        port: 3000
+      name: ip-be
+---
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: backend
+spec:
+  parentRefs:
+    - name: eg
+  hostnames:
+    - "www.example.com"
+  rules:
+    - backendRefs:
+        - group: gateway.envoyproxy.io
+          kind: Backend
+          name: backend-mixed-ip-uds
+          weight: 1
+        - group: ""
+          kind: Service
+          name: backend
+          port: 3000
+          weight: 1          
+      matches:
+        - path:
+            type: PathPrefix
+            value: /
+---
+apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: BackendTLSPolicy
+metadata:
+  name: policy-btls
+spec:
+  targetRef:
+    group: gateway.envoyproxy.io
+    kind: Backend
+    name: backend-mixed-ip-uds
+    sectionName: uds-be
+  tls:
+    caCertRefs:
+      - name: backend-tls-checks-certificate
+        group: ''
+        kind: ConfigMap
+    hostname: example.com
+```
+
+## Design Decisions
+* All existing and future `BackendObjectReference` in Envoy Gateway MUST support the `Backend` kind. 
+* Gateway-API and Envoy Gateway policies that attach to Services ([BackendTLSPolicy][], [BackendLBPolicy][]) 
```

Comment:
-1 on attaching BTS to a Backend, we technically have 4 levels of implicit defaults and overrides (`gateway`, `;listener`, `route` and `route rule` soon), adding more will further increase complexity

Reply:
How is that different from supporting this for services, though? In terms of defaults/overrides, doesn't this introduce the same amount of layers that exists today?

Reply:
as discussed, this was a misunderstanding. BackendTrafficPolicy will not attach to backend resources.